# Bin message

# Bin name
string bin_name

# Material type
string material_type

# Camera Image Information
osrf_gear/LogicalCameraImage image
